backend/audit/test_validators.py

```diff
@@ -659,7 +659,14 @@ def test_validation_is_applied(self):
         template = json.loads(template_definition_path.read_text(encoding="utf-8"))
         invalid = json.loads('{"CorrectiveActionPlan":{}}')
         expected_msg = str(
-            ("A", "2", "Auditee UEI", template["sheets"][1]["single_cells"][0]["help"])
+            [
```

Comment:
I don't understand why this happened, but these validator errors started coming back as a list (which is what I would expect given the `list()` call here: https://github.com/GSA-TTS/FAC/blob/main/backend/audit/validators.py#L156). So maybe I'm not sure why they _weren't_ lists before and which part of this set of commits changed it.

Reply:
We'll pick this up *real soon now* when we start doing more E2E testing with lots of real data.